deploy/templates/notificationtemplates/sandbox/idlertriggered/notification.html

```diff
@@ -41,22 +41,18 @@
 >
 
     <p>
-        You are receiving this email because one or more of your workloads in the Developer Sandbox for
-        Red Hat OpenShift has been running for 12 hours.
+        You are receiving this email because one or more of your workloads in the Developer Sandbox has been running for 12 hours.
     </p>
 
     <p>
-        In accordance with the usage terms of Developer Sandbox, your workload {{.AppType}} {{.AppName}} has been scaled down. You can restart your workload(s) or scale up the number of instances from the
-        Developer Sandbox User Interface.
-    </p>
-
-    <p>
-        Please reach us at {{.ReplyTo}} with any questions.
+       After 12 hours, our system idles running workloads.
+        You can <a href="https://developers.redhat.com/learn/openshift/revive-inactive-openshift-pods-scaled-zero">restart your workload,</a> {{.AppType}} {{.AppName}}
+        when you're ready to continue exploring on the Developer Sandbox.
```

Comment:
Is this multi-line format intended? Mainly splitting the second sentence into two lines?

Reply:
no, it was an identation issue. fixed now. but even with the identation wrong, the web preview wouldn't have been affected. I'm now adding web preview of the emails also for easier readability.